spec/fixtures/hiera/default.yaml

```diff
@@ -98,37 +98,46 @@ nebula::profile::puppet::master::fileservers:
 nebula::profile::puppet::master::autosign_whitelist: []
 
 nebula::users::key_file: /etc/secretkeys/invalid.default
-nebula::users::groups:
+nebula::users::default_host: invalid.default
+nebula::sudoers:
+  invalid_normal_admin:
+    type: ssh-rsa
+    key: >-
+      AAAA
+      AAAA
+      AAAA
+  invalid_special_admin:
+    type: ssh-dsa
+    host: invalid.special
+    key: >-
+      BBBB
+      BBBB
+      BBBB
+
```

Comment:
Do we need this built out this much in `default.yaml`? Seems like it might be preferable just to provide empty lists

Reply:
I'm sure I can update the tests and put that in a separate yaml file. Good eye.